apps/src/gamelab/animationLibrary.json

```diff
@@ -0,0 +1,332 @@
+      "sourceSize": {
+        "x": 150,
+        "y": 181
+      }
+    },
+    "bunny1_ready": {
+      "name": "bunny1_ready",
+      "aliases": [
+        "jumper"
+      ],
+      "frameSize": {
+        "x": 120,
+        "y": 191
+      },
+      "frameCount": 1,
+      "looping": false,
+      "frameDelay": 2,
+      "version": "bHuAAcEnoZi32fMO4YotvbwsYJXVDxlp",
+      "sourceUrl": "/api/v1/animation-library/bHuAAcEnoZi32fMO4YotvbwsYJXVDxlp/bunny1_ready.png",
+      "sourceSize": {
+        "x": 120,
+        "y": 191
+      }
+    },
+    "bunny1_stand": {
+      "name": "bunny1_stand",
+      "aliases": [
+        "jumper"
+      ],
+      "frameSize": {
+        "x": 120,
+        "y": 201
+      },
+      "frameCount": 1,
+      "looping": false,
+      "frameDelay": 2,
+      "version": "fiS3z042HzgVG4_tW.7DkMHXthq_gTd1",
+      "sourceUrl": "/api/v1/animation-library/fiS3z042HzgVG4_tW.7DkMHXthq_gTd1/bunny1_stand.png",
+      "sourceSize": {
+        "x": 120,
+        "y": 201
+      }
+    },
+    "bunny1_walk1": {
+      "name": "bunny1_walk1",
+      "aliases": [
+        "jumper"
+      ],
+      "frameSize": {
+        "x": 120,
+        "y": 201
+      },
+      "frameCount": 1,
+      "looping": false,
+      "frameDelay": 2,
+      "version": "homcthRNfapnn_m1rpoSxAYz.EbovQOZ",
+      "sourceUrl": "/api/v1/animation-library/homcthRNfapnn_m1rpoSxAYz.EbovQOZ/bunny1_walk1.png",
+      "sourceSize": {
+        "x": 120,
+        "y": 201
+      }
+    },
+    "bunny1_walk2": {
+      "name": "bunny1_walk2",
+      "aliases": [
+        "jumper"
+      ],
+      "frameSize": {
+        "x": 120,
+        "y": 207
+      },
+      "frameCount": 1,
+      "looping": false,
+      "frameDelay": 2,
+      "version": "Fda2FKgUXm1D3onUruFgxV17ozDgTY.1",
+      "sourceUrl": "/api/v1/animation-library/Fda2FKgUXm1D3onUruFgxV17ozDgTY.1/bunny1_walk2.png",
+      "sourceSize": {
+        "x": 120,
+        "y": 207
+      }
+    },
+    "ghost_standing": {
+      "name": "ghost_standing",
+      "aliases": [
+        "p5.play"
+      ],
+      "frameSize": {
+        "x": 72,
+        "y": 158
+      },
+      "frameCount": 7,
+      "looping": true,
+      "frameDelay": 2,
+      "version": "bsPQ1dVzEpXey4z7U34gVs_aJ4CmMlyr",
+      "sourceUrl": "/api/v1/animation-library/bsPQ1dVzEpXey4z7U34gVs_aJ4CmMlyr/ghost_standing.png",
+      "sourceSize": {
+        "x": 504,
+        "y": 158
+      }
+    },
+    "platform": {
+      "name": "platform",
+      "aliases": [
+        "p5.play"
+      ],
+      "frameSize": {
+        "x": 200,
+        "y": 73
+      },
+      "frameCount": 3,
+      "looping": true,
+      "frameDelay": 2,
+      "version": "nZXoIMRM6WfMyEV_Azq7Kw4_5MDE7JUp",
+      "sourceUrl": "/api/v1/animation-library/nZXoIMRM6WfMyEV_Azq7Kw4_5MDE7JUp/platform.png",
+      "sourceSize": {
+        "x": 200,
+        "y": 219
+      }
+    }
+  },
+  "aliases": {
```

Comment:
Here's the alias map that we use for searching - we search the keys, and the results are all the values of all the matched keys.

Once the library gets large, we might benefit from generating a [Trie](https://en.wikipedia.org/wiki/Trie) instead, at least to two or three characters deep.